components/home_page_search_discovery/home_page_search_discovery.react.js

```diff
@@ -0,0 +1,366 @@
+    /**
+     * Set properties / destructuring
+     */
+    const { valueShow, valueBetween, valueAnd } = this.state;
+
+    /**
+     * Error messages
+     * Hide the error message while type
+     */
+    this.hideErrorMsg(valueShow, 'errorShow', this.mainState.valueShow);
+    this.hideErrorMsg(
+      valueBetween,
+      'errorBetween',
+      this.mainState.valueBetween
+    );
+    this.hideErrorMsg(valueAnd, 'errorAnd', this.mainState.valueAnd);
+  };
+
+  onSubmitSearch = e => {
+    /**
+     * Set properties / destructuring
+     */
+    const {
+        valueShow,
+        valueAnd,
+        valueBetween,
+        errorBetween,
+        errorAnd,
+        errorShow
+      } = this.state,
+      { fieldShow, fieldBetween, fieldAnd } = this.state.Data.form;
+
+    /**
+     * Error messages
+     */
+    const errMsgIfEmptyFieldShow = this.showErrorMsgIfEmptyOrWhiteSpace(e),
+      dateValidBetween = this.dateValid(e),
+      dateValidAnd = this.dateValid(e),
+      stopForm = this.preventFormSubmission(e),
+      showErrorMsgEarlierBetween = this.showErrorMsgEarlierThan(e);
+
+    // Error Message: Between date must be earlier than and date
+    showErrorMsgEarlierBetween(
+      valueBetween,
+      valueAnd,
+      'errorBetween',
+      fieldBetween.errorMsgEarlier
+    );
+
+    // Error Message: Please enter keyword or catalogue reference
+    errMsgIfEmptyFieldShow(valueShow, 'errorShow', fieldShow.errorMsg);
+
+    // Error Message: You have entered an invalid date format
+    dateValidBetween(
+      valueBetween,
+      'errorBetween',
+      fieldBetween.errorMsgInvalid
+    );
+
+    // Error Message: You have entered an invalid date format
+    dateValidAnd(valueAnd, 'errorAnd', fieldAnd.errorMsgInvalid);
+
+    //On Form Update
+    stopForm(errorShow, errorBetween, errorAnd);
+
+    this.stateIsOnFire();
+
+    // Set the value to the hidden element
+    this.hiddenRef.current.value = this.state.Data.form.hiddenField.valueHidden;
+
+    setTimeout(() => this.pushInDataLayer(this.objGTM), 100);
+  };
+
+  // Use of React lifecycle method
+  componentDidUpdate(prevProps, prevState) {
+    this.buildGTMObj(this.objGTM);
+  }
+
+  render() {
+    const {
+        valueShow,
+        valueBetween,
+        valueAnd,
+        errorShow,
+        errorBetween,
+        errorAnd
+      } = this.state,
+      { mainHead, id, form, links } = this.state.Data,
+      {
+        formId,
+        method,
+        action,
+        role,
+        fieldShow,
+        fieldBetween,
+        fieldAnd,
+        fieldAcross,
+        inputSearch,
+        grid,
+        hiddenField
+      } = form;
+    return (
+      <Wrapper id={id}>
+        <Form
+          id={formId}
+          name={formId}
+          method={method}
+          action={action}
+          onSubmit={this.onSubmitSearch}>
+          <div className={grid.container} role={role}>
+            <div className={grid.group.headline}>
+              <h1>{mainHead}</h1>
```

Comment:
This should not be a `<h1>` tag.
When the component mounts, and if there is an `<h1>` tag, there could be a possibility of duplicates, which is another reason the `<form>` should have a `<fieldset>` and `<legend>`.